src/main/java/mandel/fallingsand/Main.java

```diff
@@ -7,6 +7,7 @@ public static void main(String[] args) {
         sand.randomSand(50);
         boolean keepFalling = true;
         while (keepFalling) {
+            new SandFrame().setVisible(true);
```

Comment:
Don't do this.
Don't mix your command line program with your GUI program. 
Also you could end up creating multiple SandFrames since this is in a loop

Reply:
Should I delete the command line program or just take the gui out of the while loop?

Reply:
Delete it